compiler/ballerina-lang/src/test/java/io/ballerina/projects/CyclicDependenciesTest.java

```diff
@@ -1,116 +1,116 @@
+                DotGraphUtils.createDependencyNodeGraph(mutableGraph);
+        Set<List<ResolutionEngine.DependencyNode>> actualCycles = dependencyGraph.findCycles();
+
+        Assert.assertNotNull(actualCycles);
+        Assert.assertEquals(actualCycles.size(), expectedCycles.size());
+        actualCycles.forEach(cycle -> Assert.assertTrue(expectedCycles.contains(cycle)));
+    }
+
+    @DataProvider(name = "provideCyclicDependencies")
+    private Object[][] provideCyclicDependencies() {
+        return new Object[][]{
+                {"case001", List.of(
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_04, PACKAGE_01),
+                        Arrays.asList(PACKAGE_01, PACKAGE_04, PACKAGE_01)
+                )},
+                {"case002", List.of(
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_03, PACKAGE_01),
+                        Arrays.asList(PACKAGE_04, PACKAGE_05, PACKAGE_04),
+                        Arrays.asList(PACKAGE_01, PACKAGE_04, PACKAGE_05, PACKAGE_03, PACKAGE_01)
+                )},
+                {"case003", List.of(
+                        Arrays.asList(PACKAGE_04, PACKAGE_02, PACKAGE_04)
+                )},
+                {"case004", List.of(
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_03, PACKAGE_01),
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_04, PACKAGE_01),
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_05, PACKAGE_01),
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_06, PACKAGE_01)
+                )},
+                {"case005", List.of(
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_01)
+                )},
+                {"case006", List.of(
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_04, PACKAGE_01),
+                        Arrays.asList(PACKAGE_02, PACKAGE_04, PACKAGE_03, PACKAGE_02),
+                        Arrays.asList(PACKAGE_01, PACKAGE_03, PACKAGE_02, PACKAGE_04, PACKAGE_01)
+                )},
+                {"case007", List.of(
+                        Arrays.asList(PACKAGE_01, PACKAGE_02, PACKAGE_03, PACKAGE_01),
+                        Arrays.asList(PACKAGE_01, PACKAGE_05, PACKAGE_02, PACKAGE_03, PACKAGE_01),
+                        Arrays.asList(PACKAGE_02, PACKAGE_03, PACKAGE_02),
+                        Arrays.asList(PACKAGE_02, PACKAGE_03, PACKAGE_04, PACKAGE_05, PACKAGE_02),
+                        Arrays.asList(PACKAGE_02, PACKAGE_03, PACKAGE_06, PACKAGE_04, PACKAGE_05, PACKAGE_02)
+                )}
+        };
+    }
+
+    private static ResolutionEngine.DependencyNode createNode(String org, String name, String version) {
+        return new ResolutionEngine.DependencyNode(PackageDescriptor.from(
+                PackageOrg.from(org), PackageName.from(name), PackageVersion.from(version)
+        ));
+    }
+}
```

Comment:
What's the actual change here? Seems above code lines seems identical

Reply:
This was to fix expected linebreaks to be 'LF' but found 'CRLF' linebreak-style issue. These are using CRLF line breaks since this was added through windows. According our new checkstyle plugin configuration we need to use LF. This can be fixed using code editor. But all lines will be changes

Reply:
Thank you for the explanation.